modules/administration-guide/partials/proc_defining-custom-branding-values-for-che-theia.adoc

```diff
@@ -14,20 +14,20 @@ Create a `product.json` file with a new name of the product, logo, description,
 [source,json,attrs="nowrap"]
 ----
 {
-    "name": "Red Hat CodeReady Workspaces",  <1>
-    "icon": "codeready-workspaces-icon.png", <2>
+    "name": "Red Hat OpenShift Dev Spaces",  <1>
+    "icon": "devspaces-icon.png", <2>
```

Comment:
png files are updated to remove the prefix entirely. See https://github.com/redhat-developer/devspaces-theia/blob/devspaces-3-rhel-8/conf/theia/branding/product.json

Perhaps instead of embedding this content, you should just link to the above file?